map/autoware_lanelet2_map_validator/test/src/test_missing_regulatory_elements.cpp

```diff
@@ -0,0 +1,245 @@
+  {
+    // Create test map
+
+    // Points for stop line
+    Point3d p0 = Point3d(getId(), 0.0, 0.0, 0.1);
+    Point3d p1 = Point3d(getId(), 0.0, 1.0, 0.1);
+    Point3d p2 = Point3d(getId(), 0.0, 2.0, 0.1);
+    // Points for traffic light
+    Point3d p3 = Point3d(getId(), 0.0, 0.0, 5.0);
+    Point3d p4 = Point3d(getId(), 0.0, 1.0, 5.0);
+    Point3d p5 = Point3d(getId(), 0.0, 2.0, 5.0);
+    // Points for crosswalk
+    Point3d p6 = Point3d(getId(), 1.0, 0.0, 0.1);
+    Point3d p7 = Point3d(getId(), 1.0, 1.0, 0.1);
+    Point3d p8 = Point3d(getId(), 1.0, 2.0, 0.1);
+    Point3d p9 = Point3d(getId(), 2.0, 0.0, 0.1);
+    Point3d p10 = Point3d(getId(), 2.0, 1.0, 0.1);
+    Point3d p11 = Point3d(getId(), 2.0, 2.0, 0.1);
+
+    // Stop line
+    LineString3d sl1 = LineString3d(getId(), {p0, p1}, sl_attr_);
+    LineString3d sl2 = LineString3d(getId(), {p1, p2}, sl_attr_);
+
+    LineString3d tl1 = LineString3d(getId(), {p3, p4}, tl_attr_);
+    LineString3d tl2 = LineString3d(getId(), {p4, p5}, tl_attr_);
+
+    // LineStrings for crosswalk
+    LineString3d cw_ls1 = LineString3d(getId(), {p6, p7});
+    LineString3d cw_ls2 = LineString3d(getId(), {p7, p8});
+    LineString3d cw_ls3 = LineString3d(getId(), {p9, p10});
+    LineString3d cw_ls4 = LineString3d(getId(), {p10, p11});
+
+    Polygon3d cw_poly1 = Polygon3d(getId(), {p7, p6, p9, p10, p7}, cw_poly_attr_);
+    Polygon3d cw_poly2 = Polygon3d(getId(), {p8, p7, p10, p11, p8}, cw_poly_attr_);
+    // Lanelets for crosswalk
+    Lanelet cw1 = Lanelet(getId(), cw_ls1, cw_ls3, cw_attr_);
+    Lanelet cw2 = Lanelet(getId(), cw_ls2, cw_ls4, cw_attr_);
+
+    // Traffic light regulatory element
+    RegulatoryElementPtr tl_reg_elem1, tl_reg_elem2;
+    tl_reg_elem1 = TrafficLight::make(getId(), tl_re_attr_, {tl1}, {sl1});
+    tl_reg_elem2 = TrafficLight::make(getId(), tl_re_attr_, {tl2}, {sl2});
+
+    // Crosswalk regulatory element
+    RegulatoryElementPtr cw_reg_elem1, cw_reg_elem2;
+    cw_reg_elem1 = Crosswalk::make(getId(), cw_re_attr_, cw1, cw_poly1, {sl1});
+    cw_reg_elem2 = Crosswalk::make(getId(), cw_re_attr_, cw2, cw_poly2, {sl2});
+
+    cw1.addRegulatoryElement(cw_reg_elem1);
+    cw2.addRegulatoryElement(cw_reg_elem2);
+
+    // Add elements to map
+    for (const auto & re : {tl_reg_elem1, tl_reg_elem2}) {
+      in_map_ptr->add(re);
+    }
+    for (const auto & cw : {cw1, cw2}) {
+      in_map_ptr->add(cw);
+    }
+  }
+  AttributeMap sl_attr_, tl_attr_, cw_attr_, cw_poly_attr_, tl_re_attr_, cw_re_attr_;
+
+private:
+};
+
+TEST_F(TestSuite, ValidatorAvailability)  // NOLINT for gtest
+{
+  std::string expected_validators_concat =
+    "mapping.crosswalk.missing_regulatory_elements,"
+    "mapping.stop_line.missing_regulatory_elements,"
+    "mapping.traffic_light.missing_regulatory_elements";
+
+  lanelet::validation::Strings validators =
+    lanelet::validation::availabeChecks(expected_validators_concat);  // cspell:disable-line
```

Comment:
Is "availabe" a typo for "available"?

Reply:
Yes, but it comes from here which I cannot touch.

https://github.com/fzi-forschungszentrum-informatik/Lanelet2/blob/507033b82d9915f086f2539d56c3b62e71802438/lanelet2_validation/include/lanelet2_validation/Validation.h#L40

Reply:
This might be from the original library. Okay.